third_party/openvino/ngraph_c_api/build_ngraph_c_api_msvc.bat

```diff
@@ -17,7 +17,7 @@
 
 @setlocal
 SETLOCAL EnableDelayedExpansion
-set "OPENVINO_DIR=%ProgramFiles(x86)%\Intel\openvino_2021"
+set "OPENVINO_VERSION=openvino_2021"
```

Comment:
@fujunwei , what's your purpose to change this file?

Reply:
The latest OpenVINO still can install in `IntelSWTools` not `intel` directory, so the two directories need to be detected.